### PR TITLE
Plant card design

### PR DESCRIPTION
**Pull request by petmetic**
The detailed plant information should display only after clicking on arrows/details.

Tests will be done in different PR.

### CODE REVIEW

File: src/app/features/plants/plantExtended.tsx [hunk 1]
```diff
@@ -1,36 +1,77 @@
...
+        <div className="flex gap-1">
+          <ArrowsDown /> Details
         </div>
-        <div className="border">
-          <p className="font-semibold">Volume of water</p>
-          <p>{plant.volume}</p>
+      </CollapsibleTrigger>
+      <div className="flex flex-col gap-2 p-3">
+        <div className="p-2">
+          <div className="">
```
Comment (jurecuhalev) [inline src/app/features/plants/plantExtended.tsx:25]: why do you need this div to be outside of `CollapsibleContent` and not inside?

**Review by jurecuhalev — APPROVED** [review]

**Comment by jurecuhalev** [comment]
You probably don't need `cn()`. Seems a bit sloppy.

File: src/app/features/plants/plant-small.tsx [hunk 2]
```diff
@@ -46,25 +47,14 @@ export function PlantSmall({
...
           <p>{watered ? "Plant is watered." : "Plant needs watering."}</p>
         </Label>
-        <div>
-          <Button onClick={onExpand}>
-            <ArrowDown /> Vew details
-          </Button>
-        </div>
-        <div>
-          {expanded ? (
-            <div>
```
Comment (simonfabcic) [inline src/app/features/plants/plant-small.tsx:61]: Is there a reason for this <div>?

File: src/app/features/plants/plantExtended.tsx (hunk 1 above)
Comment (petmetic) [reply src/app/features/plants/plantExtended.tsx]: Thanks for spotting a copied typo :)

**Review by simonfabcic — APPROVED** [review]
My first review. Ever. 👩‍🎓

File: src/app/features/plants/plant-small.tsx (hunk 2 above)
Comment (simonfabcic) [reply src/app/features/plants/plant-small.tsx:61]: Oh, yes sure... My mistake :)

**Pull request merged**